vllm_spyre/model_executor/model_loader/spyre.py

```diff
@@ -388,7 +388,7 @@ def get_num_blocks_available(self) -> int:
 
         # TODO: replace the hard coded NUM_BLOCKS_SPYRE by calling a function
         # in torch_sendnn which returns the value set by the Spyre compiler.
-        if ('granite-3.3-8b-instruct' in self.model_config.model
+        if (SpyrePlatform.is_granite_33_8b(self.model_config)
```

Comment:
I wonder if we have to account for other granite models down the road

Reply:
Very likely, given current timeframes for model support